jdi-light/src/main/java/com/epam/jdi/light/elements/base/JDIBase.java

```diff
@@ -505,21 +517,27 @@ public void actions(JFunc2<Actions, WebElement, Actions> action) {
     public void actions(JFunc1<Actions, Actions> action) {
         action.execute(actions.get()).build().perform();
     }
+
     public void actionsWithElement(JFunc2<Actions, WebElement, Actions> action) {
         action.execute(actions.get().moveToElement(get()), get()).build().perform();
     }
+
     public void actionsWithElement(JFunc1<Actions, Actions> action) {
         action.execute(actions.get().moveToElement(get())).build().perform();
     }
-    private Safe<Actions> actions = new Safe<>(() -> new Actions(driver()));
 
-    public ElementArea clickAreaType = ELEMENT.clickType;
-    public TextTypes textType = ELEMENT.getTextType;
-    public SetTextTypes setTextType = ELEMENT.setTextType;
+    private final Safe<Actions> actions = new Safe<>(() -> new Actions(driver()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)